totalRP3/Locales/ptBR.lua

```diff
@@ -1091,13 +1075,13 @@ Isto é uma |cffffff00textura de fundo|r para a descrição do seu personagem.
 O modelo escolhido define o |cffffff00layout geral e possibilidades de escrita|r para sua descrição.
 |cffff9900Apenas o modelo escolhido é visível aos outros, então você não precisa preencher todos.|r
 Uma vez que um modelo for escolhido, você pode abrir este tutorial novamente para mais ajuda sobre cada modelo.]=],
-	["REG_PLAYER_TUTO_ABOUT_MISC_1"] = [=[Esta seção possui |cffffff005 slots|r onde você pode descrever |cff00ff00as mais importantes informações sobre seu personagem|r.
+	["REG_PLAYER_TUTO_ABOUT_MISC_1"] = [=[Esta seção possui |cnGREEN_FONT_COLOR:5 slots|r onde você pode descrever |cnGREEN_FONT_COLOR:as mais importantes informações sobre seu personagem|r.
 
-Esses slots estarão visíveis na sua |cffffff00"barra de À primeira vista"|r quando alguém selecionar seu personagem.
+Esses slots estarão visíveis na sua |cnGREEN_FONT_COLOR:"barra de À primeira vista"|r quando alguém selecionar seu personagem.
 
 |cff00ff00Dica: Você pode arrastar e soltar os slots para reordená-los.|r
-Também funciona na |cffffff00"barra À primeira vista"|r!]=],
-	["REG_PLAYER_TUTO_ABOUT_MISC_3"] = "Esta seção contém |cffffff00uma lista de marcadores|r para responder muitas |cffffff00perguntas comuns que as pessoas poderiam fazer a respeito de você, seu personagem e a maneira como você o interpreta|r.",
+Também funciona na |cnGREEN_FONT_COLOR:"barra À primeira vista"|r!]=],
```

Comment:
```suggestion
Também funciona na |cnGREEN_FONT_COLOR:barra "À primeira vista"|r!]=],
```
same, not sure why this one doesn't have `de` but I don't speak portuguese.